docs/docs/userguide/monitoring/PROMETHEUS.md

```diff
@@ -0,0 +1,142 @@
+# Monitoring with Prometheus
+
+The instructions outlined on this page provide details on how you can enable [Prometheus](https://prometheus.io/){.external} monitoring on your stack with [Grafana](https://grafana.com/){.external}
+
+## 1. Install Prometheus stack
+
+!!!tip "Note"
+    * This approach will also install Grafana
+    * For the purposes of this guide the Prometheus stack will be installed with the release name `prometheus-stack`.
+
+Install [kube-prometheus-stack](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack){.external} Helm chart
+
+Fetch the repo and perform an update
+```bash
+helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+helm repo update
+```
+Now install the Prometheus stack
+```bash
+helm install prometheus-stack prometheus-community/kube-prometheus-stack
```

Comment:
Perhaps it's worth adding a custom-values.yaml that should be used because out of the box persistence isn't enabled (data is lost when the pods restart). For example, I am using the following values on EKS:

```
grafana:
  persistence:
    enabled: true
    size: 10Gi
  sidecar:
    dashboards:
      enabled: true
      label: grafana_dashboard
      labelValue: dc_monitoring
prometheus:
  prometheusSpec:
    storageSpec:
      volumeClaimTemplate:
        spec:
          accessModes:
          - ReadWriteOnce
          resources:
            requests:
              storage: 10Gi
```
This way, we can also explain where label ENVs come from in Grafana dashboard importer sidecar.

Reply:
I like it, will add this in.

Reply:
@bianchi2 is the logic for handling this `prometheus` stanza available already?

Reply:
Added the following @bianchi2 

![CleanShot 2023-05-22 at 10 16 11](https://github.com/atlassian/data-center-helm-charts/assets/409063/ac085ec8-83e3-4ed6-a4b3-a404080c50d7)